darwin/DarwinMachine.c

```diff
@@ -105,18 +105,22 @@ Machine* Machine_new(UsersTable* usersTable, uid_t userId) {
    openzfs_sysctl_init(&this->zfs);
    openzfs_sysctl_updateArcStats(&this->zfs);
 
+#if !TARGET_OS_IPHONE
    this->GPUService = IOServiceGetMatchingService(kIOMainPortDefault, IOServiceMatching("IOGPU"));
    if (!this->GPUService) {
       CRT_debug("Cannot initialize IOGPU service");
    }
+#endif
```

Comment:
When compiling out this code, please at least set `this->GPUService` to a dummy value like NULL.

Reply:
If I see this correctly, `GPUService` is not even declared for iPhone 6 with this PR.

Reply:
Given the original source in the cleanup function is wrong currently too, I'd much prefer to leave the field in and correct the existing code. That way we're not un-defining and re-defining fields based on whether we believe them to be present or not and instead deferring these checks to runtime, where they can be decided much more comprehensively. AFAICT the type used for the field should be available even when compiling for the iPhone 6.